Directory.Build.props

```diff
@@ -55,10 +48,8 @@ tools\TestAssetsDependencies\TestAssetsDependencies.csproj
 
     <Import Project="build/AzureInfo.props" />
     <Import Project="build/BuildDefaults.props" />
-    <Import Project="build/DotnetCoreSdkLKG.props" />
-    <Import Project="build/BackwardsCompatibilityRuntimes.props" />
+    <Import Project="build/SetupPreviousStage.props" />
```

Comment:
Not 100% sure on this one. SetupPreviousStage now depends on BuildDefaults, because of the architecture info, but that's not going to be available for MSBuild 14.

Reply:
MSBuild 14?

Reply:
That's what the signing project uses.